lib/galaxy/tools/data_manager/manager.py

```diff
@@ -16,7 +17,10 @@
 
 
 class DataManagers:
-    def __init__(self, app, xml_filename=None):
+    data_managers: Dict[str, 'DataManager']
```

Comment:
I haven't seen this before, does this mean the dict values need to be `DataManager` classes ?

Reply:
Yeah - and typically one would just have ``Dict[str, DataManager]`` but the string representation is used to forward reference types that have yet to be defined in the file at the point where the type declaration is used.